packages/vscode-extension/package.json

```diff
@@ -22,6 +22,7 @@
   ],
   "icon": "resources/images/logo.png",
   "main": "./dist/extension.js",
+  "activationEvents": [],
```

Comment:
This we get a complain in the new version if we don't add it

Reply:
I think it'd make sense to add an activation event for onLanguage:cypher? https://code.visualstudio.com/api/references/activation-events#onLanguage

Reply:
> Note: Beginning with VS Code 1.74.0, languages contributed by your extension do not require a corresponding onLanguage activation event declaration for your extension to be activated.

We've never needed it but I'll add it